builder/bundles.go

```diff
@@ -360,7 +365,12 @@ func resolvePackages(numWorkers int, set bundleSet, packagerCmd []string, emptyD
 			// fail in the actual install to the full chroot.
 			outBuf, _ := helpers.RunCommandOutputEnv(queryString[0], queryString[1:], []string{"LC_ALL=en_US.UTF-8"})
 
-			rpm, fullRpm := repoPkgFromNoopInstall(outBuf.String())
+			rpm, fullRpm, e := repoPkgFromNoopInstall(outBuf.String())
+			if e != nil {
+				errStr := fmt.Sprint("package dependencies not resolved for package ", bundle.Name)
```

Comment:
Was it intentional to create a new error instead of using the one from `repoPkgFromNoopInstall`? Also, I think the wording here is confusing:

"package dependencies not resolved **for package ", bundle.Name**

It sounds like the bundle is called a package.

Reply:
Instead of creating a new error variable, just wrap.
`e = errors.Wrapf(e,`